frontend/src/layout/Root.tsx

```diff
@@ -147,9 +153,7 @@ export const RootLoader = <Q extends QueryWithUserData>({
     const data = usePreloadedQuery(query, queryRef);
     const userData = useFragment(userDataFragment, data);
 
-    return (
-        <UserProvider data={userData?.currentUser}>
-            <Root nav={nav(data)}>{render(data)}</Root>
-        </UserProvider>
-    );
+    return <UserProvider data={userData?.currentUser}>
+        <Root nav={nav(data)}>{render(data)}</Root>
+    </UserProvider>;
```

Comment:
Mhpf. Throughout the codebase we use extra parens if the starting tag is not trivial.